client/src/components/calendar/CalendarPage.jsx

```diff
@@ -131,13 +131,15 @@ class CalendarPage extends React.Component {
     const { meetingSuggestions, actions, candidates } = this.props;
     const selectedSuggestion = meetingSuggestions.data[selectedOption];
     const candidateUser = candidates.find(candidate => candidate.email === selectedCandidate.email);
-    actions.createEvent(selectedSuggestion, candidateUser, required, optional);
+    const response = actions.createEvent(selectedSuggestion, candidateUser, required, optional);
```

Comment:
Not sure if it was me or not, but don't we either have to `await` or put below lines in `then` block?